myia/parser.py

```diff
@@ -138,6 +162,15 @@ def __init__(self, environment: Environment,
             globals_.update(free_vars.nonlocals)
         self.globals_ = globals_
 
+    def make_location(self, node: ast.AST) -> Location:
+        """Create a Location from an AST node."""
+        if hasattr(node, 'lineno') and hasattr(node, 'col_offset'):
+            return Location(self.url,
+                            node.lineno + self.line_offset - 1,  # type: ignore
+                            node.col_offset)  # type: ignore
+        else:
```

Comment:
Can we fail early here instead of returning `None`?

Reply:
OK.